src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Rectangle extends Figure {
+    private static final String type = "rectangle";
+    private final int width;
+    private final int height;
+
+
+    public Rectangle(int width, int height, String color) {
+        this.width = width;
+        this.height = height;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) width * height;
```

Comment:
Casting to double is unnecessary here since the multiplication of two ints will result in an int, which will be automatically promoted to a double when returned.